include/tvm/relay/transform.h

```diff
@@ -105,9 +105,11 @@ TVM_DLL Pass LazyGradientInit();
 /*!
  * \brief Fold constant expressions.
  *
+ * \param fold_qnn Whether to fold constants for QNN operations.
```

Comment:
I think we need a explanation why there are not folded by default and when should the "user" of the pass should use this.